fiboa_cli/datasets/us_usda_cropland.py

```diff
@@ -0,0 +1,96 @@
+from os.path import join, dirname
+
+from .commons.ec import load_ec_mapping
+from ..util import log
+from ..convert_utils import convert as convert_
+import pandas as pd
+
+
+SOURCES = {
+    "https://www.nass.usda.gov/Research_and_Science/Crop-Sequence-Boundaries/datasets/NationalCSB_2016-2023_rev23.zip": ["NationalCSB_2016-2023_rev23/CSB1623.gdb"]
+}
+ID = "us_usda_cropland"
+SHORT_NAME = "USDA CSB"
+TITLE = "USDA Crop Sequence Boundaries"
+DESCRIPTION = """
+The Crop Sequence Boundaries (CSB) developed with USDA's Economic Research Service, produces estimates of field boundaries, crop acreage, and crop rotations across the contiguous United States. It uses satellite imagery with other public data and is open source allowing users to conduct area and statistical analysis of planted U.S. commodities and provides insight on farmer cropping decisions.
+
+NASS needed a representative field to predict crop planting based on common crop rotations such as corn-soy and ERS is using this product to study changes in farm management practices like tillage or cover cropping over time.
+
+CSB represents non-confidential single crop field boundaries over a set time frame. It does not contain personal identifying information. The boundaries captured are of crops grown only, not ownership boundaries or tax parcels (unit of property). The data are from satellite imagery and publicly available data, it does not come from producers or agencies like the Farm Service Agency.
+"""
+
+EXTENSIONS = ["https://fiboa.github.io/crop-extension/v0.1.0/schema.yaml"]
+PROVIDERS = [
+    {
+        "name": "United States Department of Agriculture",
+        "url": "https://www.nass.usda.gov/",
+        "roles": ["licensor", "producer"]
+    }
+]
+LICENSE ={
+    "title": "The USDA NASS Crop Sequence Boundaries and the data offered at https://www.nass.usda.gov/Research_and_Science/Crop-Sequence-Boundaries are provided to the public as is and are considered public domain and free to redistribute. Users of the Crop Sequence Boundaries (CSB) are solely responsible for interpretations made from these products. The CSB are provided 'as is' and the USDA NASS does not warrant results you may obtain using the data. Contact our staff at (SM.NASS.RDD.GIB@usda.gov) if technical questions arise.",
```

Comment:
The title should be short and consise. If the license can be read as such on the site, you can just use the page title:

```suggestion
    "title": "License and Liability",
```